docs/guide/tft.md

```diff
@@ -2,7 +2,7 @@
 
 Transform is available as a standalone library.
 
--   [Getting Started with TensorFlow Transform](/tfx/transform/get_started)
+-   [Getting Started with TensorFlow Transform](https://www.tensorflow.org/tfx/data_validation/get_started/)
```

Comment:
Thank you for pointing this out.
Shouldn't it be https://www.tensorflow.org/tfx/transform/get_started instead?

Reply:
Sorry, it's my bad that's the right one